docs/concepts/local-remote.md

```diff
@@ -0,0 +1,67 @@
+(local-remote)=
+# Local vs. remote workflows
+
+To run multiple functions, one after the other such as: `jobs`, `serving` and `nuclio`, you can use a kfp pipeline. There are three types of pipeline engines:
+- [Remote on KFP](#remote-kfp)
+- [KFP](#kfp)
+- Local &mdash; Used to run local pipeline with local functions, mainly for testing. Use (set `local=True in function.run()`).
+
+<img src="../_static/images/pipelines-flow.png" width="800" >
+
+All three types are configured by the `engine` flag, when running the workflow, see {py:class}`mlrun.projects.MlrunProject.run`.
+ 
+## Remote-KFP 
+
+Remote workflows are run on a pod named `workflow-runner-<workflow-name>`. This pod is responsible for loading the files from the remote source (git, tar.gz or zip) and running the KFP by using the files from the remote source.
+In some cases you might not want to load the files from the remote source, but instead use the files within the running image. See details in [build image](../projects/run-build-deploy.md#build_image),. In this case, you need to build an image that contains the workflow file and then change the workflow runner source to the local path in the image. See the example below.
+
+You can modify the pod image, source, and the pod node selector with:
+- `project.set_workflow(name="main",workflow_path="workflow.py,image="<runner-image>")` &mdash; changing the runner image
+- `project.run("main",engine="remote",workflow_runner_node_selector={"key":"value"})` &mdash; changing the runner node selector 
+- `project.run(source=<source-URL>)` &mdash; changing the runner source
+
+**This pod must be based on the image `mlrun/mlrun-kfp`. See {ref}`images-usage`.**
```

Comment:
maybe worth adding to this line a note annotations ? wdyt

Reply:
added